addons/external_intercom/XEH_preInit.sqf

```diff
@@ -0,0 +1,61 @@
+#include "script_component.hpp"
+
+#include "XEH_PREP.sqf"
+
+[
+    "TFAR_setting_externalIntercomWirelessHeadgear",
+    "EDITBOX",
+    [LSTRING(SETTING_WIRELESS_HEADGEAR_HEADER), LSTRING(SETTING_WIRELESS_HEADGEAR_DESC)],
+    [localize ELSTRING(settings,global), localize LSTRING(SETTINGS_CATEGORY_EXTERNAL_INTERCOM)],
+    ["", false, FUNC(settingForceAlphaNumericArray)],
+    1,
+    {
+        if (isServer) then {
+            TFAR_externalIntercomWirelessHeadgear = ((configFile >> "TFAR_External_Intercom_Wireless_Headgear") call BIS_fnc_getCfgSubClasses) + ([_this] call FUNC(parseClassnamesInput));
+            publicVariable "TFAR_externalIntercomWirelessHeadgear";
+        };
+    }
+] call CBA_Settings_fnc_init;
+
+[
+    "TFAR_externalIntercomEnable",
+    "LIST",
+    [LSTRING(SETTING_ENABLE_FOR_HEADER), LSTRING(SETTING_ENABLE_FOR_DESC)],
+    [localize ELSTRING(settings,global), localize LSTRING(SETTINGS_CATEGORY_EXTERNAL_INTERCOM)],
+    [[0, 1, 2], [LSTRING(SETTING_ENABLE_0), LSTRING(SETTING_ENABLE_1), LSTRING(SETTING_ENABLE_2)], 0],
+    1,
+    {
+        _vehicle = ACE_Player getVariable "TFAR_ExternalIntercomVehicle";
+        if (_this > 0 && !isNil "_vehicle") then {
```

Comment:
But if you disable it mid game, this doesn't disconnect you?

Reply:
Well.... in testing it has. Why wouldn't it? @dedmen